apps/studio/tests/global-setup.ts

```diff
@@ -0,0 +1,155 @@
+import type { StartedTestContainer } from "testcontainers"
+import { stringify } from "superjson"
+import { GenericContainer, Wait } from "testcontainers"
+import { z } from "zod"
+
+// Specify necessary container configurations here
+const configurations: ContainerConfiguration[] = [
+  {
+    name: "database",
+    image: "postgres:15-alpine",
+    ports: [5432],
+    environment: {
+      POSTGRES_USER: "root",
+      POSTGRES_PASSWORD: "root",
+      POSTGRES_DB: "test",
+    },
+    wait: { type: "PORT" },
+  },
+]
+
+export const CONTAINER_INFORMATION_SCHEMA = z.array(
```

Comment:
hmm, just wondering if we should do  `Record<name, this_schema>` as we have a small universe of containers and having the name makes it clearer to refer to!